internal/guest/linux/ioctl.go

```diff
@@ -46,3 +46,10 @@ func Ioctl(f *os.File, command int, dataPtr unsafe.Pointer) error {
 	}
 	return nil
 }
+
+// Ported from _IOWR macro.
+// Returns value for `command` parameter in Ioctl().
```

Comment:
We can't have this function as it may be construed as a derived work.

Reply:
Hard wire the results. We will get rid of the 5.15 path very soon.

Reply:
Done. Tested it again in in both 5.x and 6.x.